src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/TimeoutTests.cs

```diff
@@ -318,7 +318,7 @@ public static void AiaFetchTimeout()
             using (X509Certificate2 rootCert = rootAuthority.CloneIssuerCert())
             using (X509Certificate2 intermediateCert = intermediateAuthority.CloneIssuerCert())
             {
-                TimeSpan delay = TimeSpan.FromSeconds(3);
+                TimeSpan delay = TimeSpan.FromSeconds(10);
```

Comment:
Does this make the test take 10 seconds?  If yes it should be outerloop.

Reply:
The whole class is outer loop. 

https://github.com/dotnet/runtime/blob/18dddf19bdb2bf5a8e0da05d9307a91e3c69f55a/src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/TimeoutTests.cs#L11

Reply:
Ah. Good :)